### PR TITLE
Cleanup readme and update how mission statement is presented in repo

### PR DESCRIPTION
**Pull request by dcwalk**
This:

- removes all but the approved mission statement from the repo
- makes clear the weekly roundtables are still paused
- adds the quarterly coordinating calls
- remove `STRATEGY.md`, `model.md`, `problem.md` from repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ We have a public calendar that lists Data Together events and a Youtube playlist
 📅 [**Data Together Calendar**](https://calendar.google.com/calendar/embed?src=u75o4fbnv59006peo07nv67vsg%40group.calendar.google.com&ctz=America%2FToronto)  
 ▶️ [**Call Playlist**](https://www.youtube.com/playlist?list=PLtsP3g9LafVul1gCctMYGm9sz5FUWr5bu)
 
-### Weekly Roundtables
+### ~~Weekly Roundtables~~
```
Comment (lightandluck) [inline README.md]: Would it help to explicitly say that these are on hold?

Comment (lightandluck) [reply README.md]: Oh, I missed that. Let's merge it 👍

Comment (dcwalk) [reply README.md]: Ehhh, thinking more about this I think you are right @lightandluck and we want to be more current-- I think I read too quickly before, as this seems to imply that we might be returning to a regular schedule.

Comment (dcwalk) [reply README.md]: Updated above @lightandluck ! What do you think?

Comment (lightandluck) [reply README.md]: Looks great! Thanks for updating all of this 🙏

**Comment by dcwalk** [comment]
Okay I'm gonna jump on merging this, we can revisit/ensure these updates reflects all partners perspectives at the Fall Coordinating Call!

**Pull request merged**